libs/permit-utils/src/lib/getPermitUtilsInstance.ts

```diff
@@ -33,7 +33,7 @@ export function getPermitUtilsInstance(
 
   // TODO: allow to receive the signer as a parameter
   const web3ProviderConnector = new PermitProviderConnector(provider, account ? undefined : PERMIT_SIGNER)
-  const eip2612PermitUtils = new Eip2612PermitUtils(web3ProviderConnector)
+  const eip2612PermitUtils = new Eip2612PermitUtils(web3ProviderConnector, { enabledCheckSalt: true })
```

Comment:
Could this break the check for any existing token that doesn't support the salt being enabled?

Reply:
Then, for the token-lists repo, let's force update DAI on Polygon too

https://github.com/cowprotocol/token-lists/blob/73a0f033a5ed5349ef9ac5731d7c926e090ffd0b/src/public/PermitInfo.137.json#L2219-L2221